docs/containers/python-configure-containers.md

```diff
@@ -137,41 +136,69 @@ Exception has occurred: PermissionError
 To solve this issue, we need to correctly add permissions to the non-root user to gain access to this specific file or directory in the container. Within your Dockerfile, add:
 
 ```dockerfile
+# Creates a non-root user and adds permission to access the /app folder
 RUN useradd appuser && chown -R appuser /app
 
-# Adds permission to appuser (non-root) for access to the /extra folder
+# Adds permission for appuser (non-root) to access the /extra folder
 RUN chown -R appuser /extra
 ```
 
-> **Note**: This is just one example of how to add permissions. There are many ways to do so and it is your responsibility give the least permission possible to specific files and folders.
+> **Note**: This is just one example of how to add permissions in a container. There are many ways to do so, and it is your responsibility give the least permission possible to specific files and folders.
 
-### Binding to an inaccessible port
+### Invalid file permission on the host (Linux)
 
-If your container starts and stops immediately after `kb(workbench.action.debug.start)` without producing logs in the **Debug Console**, we can try to diagnose the issue through these steps:
+In the previous example, we showed you how to add permissions to a file or folder on the container as a non-root user. However, if you are trying to access a folder **on the host machine** from within the container as a non-root user, the user ID or group ID in the container must have access to the files on the host. To solve this issue in Linux, you might need to set file access control lists (setfacl).
+
+If you have a folder named `/share` on your host machine and try to access this folder before the access control list are properly set, you will likely receive this error:
+
+```python
+PermissionError: [Errno 13] Permission denied: '/share/logs/log.txt'
+```
+
+In order to give access to a non-root user `appuser` from within the container, follow these steps:
+
+1. From the container command line, run one of these commands:
+
+    ```powershell
```

Comment:
why powershell and not shell/bash?

Reply:
same for line 173

Reply:
That's just telling the docs site how to syntax highlight this chunk of code. Those commands are actually shell/bash commands.

Reply:
Yeah I know, this might seem a little petty but why not mark the block specifically as `bash`/`sh`? I'm pretty sure whichever markdown engine is used, it supports this language.

Reply:
... or  `shell`  I agree with @TheYarin